### PR TITLE
Add summary to package data

### PR DESCRIPTION
**Pull request by semtexzv**

### CODE REVIEW

File: manager/controllers/packages.go [hunk 1]
```diff
@@ -30,15 +31,15 @@ type PackagesResponse struct {
 }
 
 func packagesQuery(acc string) *gorm.DB {
-	return database.Db.
+	return database.Db.Debug().
```
Comment (Josca) [inline manager/controllers/packages.go:34]: It's forgotten Debug() I guess

File: manager/controllers/packages.go [hunk 2]
```diff
@@ -16,11 +16,12 @@ var PackagesOpts = ListOpts{
 	DefaultSort:    "name",
 }
 
+// nolint: lll
 type PackageItem struct {
 	Name             string `json:"name" query:"pn.name"`
 	SystemsInstalled int    `json:"systems_installed" query:"count(sp.id)"`
 	SystemsUpdatable int    `json:"systems_updatable" query:"count(sp.id) filter (where spkg.update_data is not null)"`
-	// Description      string `json:"description"`
+	Summary          string `json:"summary" query:"(select s.value from package p inner join strings s on p.summary_hash = s.id inner join advisory_metadata am on p.advisory_id = am.id where p.name_id = pn.id order by am.public_date limit 1)"`
```
Comment (Josca) [inline manager/controllers/packages.go:24]: Funny solution, ok it's working. Is it at least possible to split the query text into multiple lines to be more clear?

**Review by Josca — CHANGES_REQUESTED** [review]

File: manager/controllers/packages.go (hunk 2 above)
Comment (semtexzv) [reply manager/controllers/packages.go:24]: Tried, however it wasn't accepted by the parser for `query` attribute

**Comment by Josca** [comment]
pipeline failed

**Review by Josca — APPROVED** [review]

**Pull request merged**